### ISSUE TITLE
Can python packages processed by Grumpy be used from Go?

### ISSUE THREAD

**Issue by kulshekhar** [google/grumpy#10]
Is it possible to use Grumpy to convert eligible python packages for use from Go?

**Comment by trotterdylan** [comment]
That is an important use case and I plan on supporting it. I thought it was possible as-is but I realized there's one missing piece: grumpy.NewRootFrame() does not exist. I'll add that function shortly. In theory, this is how you would accomplish it (assuming PWD is the checked out source dir):

$ make
$ cat > hello.py
def foo(): 
  print 'hello'
^D
$ mkdir build/src/hello
$ PYTHONPATH=$PWD/build/lib/python2.7/site-packages tools/grumpc -modname=hello hello.py
$ cat > main.go
package main
import (
	"hello"
	"grumpy"
)
func main() {
	f := grumpy.NewRootFrame()
	mods, _ := grumpy.ImportModule(f, "hello", []*grumpy.Code{hello.Code})
	foo, _ := grumpy.GetAttr(f, mods[0], grumpy.NewStr("foo"), nil)
	foo.Call(f, nil, nil)
}
^D
$ GOPATH=$PWD/build go build main.go

The meat of it is that you're importing the generated "hello" Go package, passing the hello.Code handle into grumpy.ImportModule() to execute the body of the module, getting the "foo" attribute (the function foo from hello.py), and then calling foo().

Apologies for this being so rough around the edges. This is not actually a use case I've tried myself. Will update this issue once NewRootFrame() is implemented.
